content/doc/tutorials/download-and-run-jenkins-in-docker.adoc

```diff
@@ -0,0 +1,137 @@
+. Create the following Docker data volumes that will be used to persistently
+store all data in the Jenkins home directory and Maven repository (explained in
+more detail later) by running the commands: +
+  `docker volume create jenkins-data` +
+  and +
+  `docker volume create maven-repository`
++
+--
+NOTE: These data volumes allow you to shut down Docker (for example, part way
+through the tutorial) without you losing any work you've already done.
+--
++
+. Start the downloaded Jenkins/Blue Ocean Docker image (for running this
+tutorial) using the appropriate command:
+* *On Mac OS X:*
++
+----
+docker run --rm \
+  --name jenkins-tutorials \
+  -u root \
+  -p 8080:8080 \
+  -v jenkins-data:/var/jenkins_home \
+  -v maven-repository:/root/.m2 \
+  -v /var/run/docker.sock:/var/run/docker.sock \
+  -v /Users:/Users \
+  jenkinsci/blueocean
+----
++
+* *On Linux:* +
+  Use the Mac OS X command above but replace `-v /Users:/Users` with
+  `-v /home:/home`
+* *On Windows:* +
+  Use the Mac OS X command above but replace `-v /Users:/Users` with
+  `-v c:/Users:/Users` (assuming that the user's home directory is on the C
+  drive). Also, if you're using the Windows command prompt, replace the
+  backslash symbols `\` at the end of each line with carat symbols instead `^`,
+  giving you:
++
+----
+docker run --rm ^
+  --name jenkins-tutorials ^
+  -u root ^
+  -p 8080:8080 ^
+  -v jenkins-data:/var/jenkins_home ^
+  -v maven-repository:/root/.m2 ^
+  -v /var/run/docker.sock:/var/run/docker.sock ^
+  -v c:/Users:/Users ^
+  jenkinsci/blueocean
+----
+
++
+[NOTE]
+====
+This command's options do the following:
+
+* `--rm` - Automatically removes the Docker container (which is the
+  instantiation of this Jenkins/Blue Ocean Docker image) when the Docker image
+  is shut down. Keeps things tidy if you need to quit Jenkins.
+* `--name jenkins-tutorials` - Assigns the name “jenkins-tutorials” to the
+  container of the Jenkins/Blue Ocean Docker image. Makes it easier to
+  identify the container in Docker.
+* `-u root` - Sets the default user within the container to be root. This is not
+  a recommended practice for running Docker images in production. However, this
+  is fine for the sake of this tutorial.
+* `-p 8080:8080` - Makes the container running the Jenkins/Blue Ocean Docker
+  image accessible through port 8080.
+* `-v jenkins-data:/var/jenkins_home` - Mounts the `jenkins-data` data volume
+  (which you created in the previous step) with the `/var/jenkins_home`
+  directory in the container.
+* `-v maven-repository:/root/.m2` - Mounts the `maven-repository` data volume
+  with the `/root/.m2` directory in the container.
+* `-v /Users:/Users` (and its variants for other operating systems) - Mounts the
+  `/Users` directory on your local machine to a directory called `/Users` in the
+  container.
+====
+
++
+. After the 2 sets of asterisks appear in the terminal/command prompt window,
```

Comment:
It might be beneficial to include a screenshot or two in this section so the reader can compare and contrast what they see on the screen.

Reply:
Sure - will do.